### PR TITLE
Update managing-a-custom-domain-for-your-github-pages-site.md

### PR DESCRIPTION
**Pull request by JOJ0**
This change tries to clarify that one DNS A record always points to just one IP address. The current docs state that _one_ A record should be created that is pointing to multiple IP addresses.

### CODE REVIEW

**Review by tmetz — DISMISSED** [review]
Thank you so much for taking the time to submit these PRs @JOJ0!  I'll mostly defer to our docs team, but just have one small suggestion.

**Comment by janiceilene** [comment]
@JOJ0 Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by lucascosti — APPROVED** [review]
Thanks, @JOJ0 and @tmetz! I'll get this merged in 🎉

**Comment by bolek193** [comment]
_Sent from my Galaxy S10 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: content/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ To set up a `www` or custom subdomain, such as `www.example.com` or `blog.exampl
 
 ### Configuring an apex domain
 
-To set up an apex domain, such as `example.com`, you must configure a _CNAME_ file  in your {% data variables.product.prodname_pages %} repository and an `ALIAS`, `ANAME`, or `A` record with your DNS provider.
+To set up an apex domain, such as `example.com`, you must configure a _CNAME_ file  in your {% data variables.product.prodname_pages %} repository and an `ALIAS`, `ANAME`, or multiple `A` records with your DNS provider.
```
[the remark this answers is not in the document]
Comment (JOJ0) [reply content/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site.md]: hi, yes that's even better an actually more technically correct! thanks!

**Comment by JOJ0** [comment]
thanks a lot for taking care of this! :-)

**Pull request merged**